src/handwritten/system/storage/persisted_channel.py

```diff
@@ -16,12 +16,14 @@ class PersistedChannel:
     """
     __slots__ = ['_name', '__weakref__']
 
-    def __init__(self, name):
+    def __init__(self, name, interpreter):
```

Comment:
This is a public constructor. Adding a required interpreter parameter breaks compatibility with client code that directly constructs it like `nidaqmx.system.storage.PersistedChannel("MyChannel")`.

See my comments on `scale.py.mako`.